pyproject.toml

```diff
@@ -64,6 +64,28 @@ select = [
     "PLE", #pylint
 ]
 
+extend-select = [
+    "F401",    # unused-import
+    "F841",    # Unused variable
+    "W291",    # Trailing whitespace (low-hanging fruit that improves cleanliness)
+    "D419",    # Empty docstring (suggests bad documentation hygiene.)
+    "PLW0127", # Self-assigning variable
+    "PLW0128", # Redeclared assigned name
+    "UP004",   # Useless object inheritance
+    "PLE0101", # return in __init__ – breaks instantiation logic.
+    "PLE0704", # Misplaced bare raise – makes debugging harder.
+    "PLE1142", # Await outside async function – common async error.
```

Comment:
Individual PLE rules don't need to be added to `extend-select`, we are already selecting `PLE` as a whole in `select`